packages/vats/src/orchestration.js

```diff
@@ -0,0 +1,268 @@
+// @ts-check
+/** @file Orchestration service */
+import { makeTracer } from '@agoric/internal';
+import { V as E } from '@agoric/vat-data/vow.js';
+import { M, matches } from '@endo/patterns';
+import { AmountShape, BrandShape } from '@agoric/ertp';
+import { makeICAConnectionAddress, parseAddress } from '@agoric/orchestration';
+import '@agoric/network/exported.js';
+
+const { Fail, bare } = assert;
+const trace = makeTracer('Orchestration');
+
+// XXX improve me
```

Comment:
more of a TODO, enhancement. there's nothing untoward about this to be considered tech debt.